docs/index.md

```diff
@@ -153,6 +153,14 @@ like any other profile if stored in `$HOME/.config/retis/profiles`. Here is an
 profile](https://github.com/retis-org/retis/blob/main/retis/test_data/profiles/example.yaml)
 with inlined comments. If a profile is generic enough, consider contributing it!
 
+A custom directory containing profiles can be used. This is handy for sharing
+out-of-tree profiles.
```

Comment:
If the goal is to share out-of-tree profiles, why not adding the directory to the profile list instead of replacing it? something like "--extra-profile-dir" so "generic" and all other in-tree ones can also be accessed.